examples/digital_fingerprinting/production/morpheus/dfp/utils/dfp_arg_parser.py

```diff
@@ -35,6 +35,18 @@ class TimeFields:
 
 class DFPArgParser:
 
+    class Decorators:
```

Comment:
I dont think we need a nested class here. Just put it in the `DFPArgParser` class as a `classmethod`